projectfiles/Xcode/Wesnoth.xcodeproj/project.pbxproj

```diff
@@ -2482,8 +2478,8 @@
 		EC5590761A0B11C600675179 /* lua_map_generator.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = lua_map_generator.cpp; sourceTree = "<group>"; };
 		EC59F25F1A4529D2001910CB /* context.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; name = context.cpp; path = plugins/context.cpp; sourceTree = "<group>"; };
 		EC59F2601A4529D2001910CB /* manager.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; name = manager.cpp; path = plugins/manager.cpp; sourceTree = "<group>"; };
-		EC5C242218EF07B4001FA499 /* libbz2.1.0.dylib */ = {isa = PBXFileReference; lastKnownFileType = "compiled.mach-o.dylib"; name = libbz2.1.0.dylib; path = lib/libbz2.1.0.dylib; sourceTree = "<group>"; };
-		EC5C242318EF07B4001FA499 /* libexpat.1.dylib */ = {isa = PBXFileReference; lastKnownFileType = "compiled.mach-o.dylib"; name = libexpat.1.dylib; path = lib/libexpat.1.dylib; sourceTree = "<group>"; };
+		EC5C242218EF07B4001FA499 /* libbz2.1.0.dylib */ = {isa = PBXFileReference; lastKnownFileType = "compiled.mach-o.dylib"; name = libbz2.1.0.dylib; path = /usr/lib/libbz2.1.0.dylib; sourceTree = "<absolute>"; };
```

Comment:
I wouldn't do it this way.
* If the library is already in OS X 10.7, we don't need to reference it in the project at all. (my bet is on this case)
* If it isn't or its version is incompatible, we should bundle our dylibs.

I wonder, can we find OS X 10.7 around to try?..

Reply:
You must include them for linker. They won't be copied to .app bundle anymore (see 520-538) but we must 'include' them as absolute path for linker, don't we?

Reply:
It is doesn’t mean to fix anything. It is more like removing redundant entries.

Ho it worked before:
You downloaded Mac Compile Stuff from sourcesforge whoch provided these 4 system libraries. I don’t know from where but it definetly worked. These system dylibs were copied to .app bundle.

After mine FixXcodeDependencies:
I just symlinked these 4 system dylibs and used them like before, but: These libreries were also copied to bundle but it was for nothing, because system libs were linked as absolute path to system, what is much better behaviour than before, but dylibs were still copied to bundle, so... useless operation...

This commit:
Same features as with mine FixXcodeDependencies, but we are not copying dylibs into bundle anymore...


I found all macOS’ SDKs since 10.3 and these four libs are available since 10.4, so it should be definitely compatible with 10.7 and higher. Today I’ll test it in 10.7 VM, so I’ll be able to prove it and confirm it.

Reply:
I understand that it's not _intending_ to fix anything, but I'm wondering if it might've fixed that crash _incidentally_ - after some discussion between me and @ancestral we came to the conclusion that it was likely caused by binary compatibilities between certain libraries, and in particular I'm fairly sure libiconv was one of the libraries we singled out as the possible bad one.

Reply:
For the record, the crash I'm referring to is @2094.

Reply:
Yes I get it. I found some other problems so right after mine suicidal tendencies end then I'll make another commit. New problems aren't directly related to these 4 system libs but because of it can't test it right now. EOF

Reply:
Whoops, used @ instead of # there. Fixed.